styles/styles.css

```diff
@@ -677,16 +680,14 @@ main .section.small-vertical-margins {
     padding-top: 32px;
 }
 
-.copd-resources main .default-content-wrapper {
+.copd-resources main .default-content-wrapper,
+.getting-started main .default-content-wrapper {
     line-height: 1.2em;
     font-size: 1.4rem;
 }
 
-.copd-resources main .default-content-wrapper p {
-    line-height: 1.2em;
-}
-
-.copd-resources main .default-content-wrapper h2 {
+.copd-resources main .default-content-wrapper h2,
+.getting-started main .default-content-wrapper h2 {
```

Comment:
this will touch everything inside the page, will change the heading of the h2 from fragments or other blocks that can be included in the getting started page

Reply:
not sure how it would be better to approach this, because the same issue should be displayed on `copd-resources` page
maybe it would be better to specify the values in the isi fragment h2, in order to handle easily the h2 from the rest of the page, but this will require that on all other framgents or blocks we should specify sizes, margins and paddings on the elements

Reply:
font size for the `.getting-started main .default-content-wrapper h2` should be `23px`
margin-top: 25px and margion-bottom: 26px, the spacing between the h2 and the paragraph should be a bit bigger